docs/en/deploy/Membership-Alluxio.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: global
+title: Alluxio Cluster Membership Management
+---
+
+In Alluxio 2.x, masters served the purpose of keeping a view of workers list and liveness. As of 304, Membership module is introduced to replace worker-registration on master.
+Membership module offers the following way of managing worker members. 
+
+1) **STATIC** - uses a static config file(default is $ALLUXIO_HOME/conf/workers) to configure a list of workers hostnames to form the alluxio cluster, it doesn't provide membership capability as to track any new member joining / leaving, member liveliness. It's merely used as a simple quickstart deployment way to spin up a DORA alluxio cluster.
+2) **ETCD** - uses a pre-configured standalone etcd cluster to manage worker membership. On startup, worker will register itself to etcd, and then keeping its liveness to etcd throughout its process lifetime.
+
+These are the capability two of these modules can do:
+|                             | STATIC  |  ETCD  |
+| ----------------------------| ------- | ------ |
+| Get full worker list        |  YES    |  YES   |
+| Get live/failed worker list |  NO     |  YES   |
+
+
+## Deployment
+
+### STATIC
+Use a static file, following the format of conf/workers.
+Append the hostname of each node into conf/workers. Append the hostname of each worker node to a new line into conf/workers. Comment out localhost if necessary. For example,
+```
+# An Alluxio Worker will be started on each of the machines listed below. 
+# localhost
+ec2-1-111-11-111.compute-1.amazonaws.com
+ec2-2-222-22-222.compute-2.amazonaws.com
+```
+And configure the alluxio-site.properties with:
+``` 
```

Comment:
for the commands that are properties-related, can you change the ``` to ```properties, thanks! should be like 31, 36, 60